setup.cfg

```diff
@@ -0,0 +1,12 @@
+[flake8]
+exclude =
+    .git,
+    __pycache__,
+    venv,
+    dist,
+    build
+max-line-length = 99
+
+[pycodestyle]
+max-line-length = 99
```

Comment:
79 or 99?

Reply:
Deliberately change this to `99` from the default value (79).

I am fine to stick on `79`